src/fiat/bank-authorities.md

```diff
@@ -7,35 +7,34 @@ title: Bank Authorities
 # Bank Authorities
 {:.no_toc}
 
-A bank authority represents an individual's consent for Centrapay to transfer
-funds to and from a bank account. In order to move funds between a bank account
-and a Centrapay wallet, there must be a verified bank authority for the
-Centrapay account that the wallet belongs to.
+A bank authority represents an individual's consent for Centrapay to transfer funds to and from a
+bank account.
+
+In order to move funds from a Centrapay wallet and a bank account (withdrawl), we only need the
```

Comment:
from a Centrapay wallet **to** a bank account

Reply:
That does read better, a fix is in the pipe.